lib/client.ts

```diff
@@ -67,23 +72,28 @@ function request(url: string, payload: any, method: string): Function {
 export class Client {
   private readonly host: string;
   private authorize: any;
+  private fetchConfig: RequestInit = {};
 
   constructor(authorizer: Authorizer, host: string = hostname) {
     this.authorize = authorizer.authorize
     this.host = host
```

Comment:
Thoughts and adding to the constructor? Set the default to what is currently otherwise provide through the constructor function?